content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -153,202 +153,94 @@ sleep-776b7bcdcd-bhn9m   2/2       Terminating   0          2m
 
 {{< text yaml >}}
-apiVersion: v1
-kind: ConfigMap
+apiVersion: operator.istio.io/v1alpha1
+kind: IstioOperator
 metadata:
-  name: istio-sidecar-injector
-data:
-  config: |-
-    policy: enabled
-    neverInjectSelector:
-      - matchExpressions:
-        - {key: openshift.io/build.name, operator: Exists}
-      - matchExpressions:
-        - {key: openshift.io/deployer-pod-for.name, operator: Exists}
-    template: |-
-      initContainers:
-...
-{{< /text >}}
-
-The above statement means: Never inject on pods that have the label `openshift.io/build.name` **or** `openshift.io/deployer-pod-for.name` – the values of the labels don't matter, we are just checking if the keys exist. With this rule added, the OpenShift Builds use case illustrated above is covered, meaning auxiliary pods will not have sidecars injected (because source-to-image auxiliary pods **do** contain those labels).
-
-For completeness, you can also use a field called `alwaysInjectSelector`, with similar syntax, which will always inject the sidecar on pods that match that label selector, regardless of the global policy.
-
-The label selector approach gives a lot of flexibility on how to express those exceptions. Take a look at [these docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) to see what you can do with them!
-
-{{< tip >}}
-It's worth noting that annotations in the pods have higher precedence than the label selectors. If a pod is annotated with `sidecar.istio.io/inject: "true/false"` then it will be honored. So, the order of evaluation is:
-
-`Pod Annotations → NeverInjectSelector → AlwaysInjectSelector → Default Policy`
-{{< /tip >}}
-
-#### Uninstalling the automatic sidecar injector
-
-{{< text bash >}}
-$ kubectl delete mutatingwebhookconfiguration istio-sidecar-injector
-mutatingwebhookconfiguration.admissionregistration.k8s.io "istio-sidecar-injector" deleted
-{{< /text >}}
-
-The above command will not remove the injected sidecars from Pods. A
-rolling update or simply deleting the pods and forcing the deployment
-to create them is required.
-
-Optionally, it may also be desirable to clean-up other resources that
-were modified in this task.
-
-{{< text bash >}}
-$ kubectl label namespace default istio-injection-
-namespace/default labeled
-{{< /text >}}
+  name: istio
+spec:
+  values:
+    sidecarInjectorWebhook:
+      templates:
+        custom: |
+          spec:
+            containers:
+            - name: istio-proxy
+              env:
+              - name: GREETING
+                value: hello-world
+{{< /text >}}
+
+Pods will, by default, use the `sidecar` injection template, which is automatically created.
+This can be overridden by the `inject.istio.io/templates` annotation.
+For example, to apply the default template and our customization, you can set `inject.istio.io/templates=sidecar,custom`.
```

Comment:
sorry this example is confusing to me.  I'm not sure what the `sidecar,custom` mean here. would be nice to give out a consistent example as the name GREETING value hello-world earlier.

Reply:
There is a default `sidecar` template, which injects the sidecar. Additionally, user defines a `custom` template. They then tell the injector to use both of these. It will apply the sidecar template (injection istio-proxy container), then add the `custom` template (injecting GREETING env var)

Reply:
I see, this is an annotation on the pod level to enable the default + custom injection template.